libs/logic-apps-shared/src/designer-client-services/lib/standard/run.ts

```diff
@@ -101,9 +110,22 @@ export class StandardRunService implements IRunService {
   async getRun(runId: string): Promise<Run> {
     const { apiVersion, baseUrl, httpClient, workflowName } = this.options;
 
-    const uri = `${baseUrl}/workflows/${workflowName}/runs/${runId}?api-version=${apiVersion}&$expand=properties/actions,workflow/properties`;
+    let uri = `${baseUrl}/workflows/${workflowName}/runs/${runId}?api-version=${apiVersion}&$expand=properties/actions,workflow/properties`;
 
     try {
+      if (uri.toLowerCase().includes('microsoft.app')) {
```

Comment:
Should we use the hybrid app check method here?  It looks like that's what we're trying to do

Reply:
yep you're right, let me change that